lib/PAUSE/dist.pm

```diff
@@ -9,6 +9,7 @@ use List::MoreUtils ();
 use PAUSE ();
 use Parse::CPAN::Meta;
 use PAUSE::mldistwatch::Constants;
+use JSON::XS ();
```

Comment:
We should not use JSON::XS directly in toolchain code -- we already have a wrapper around various JSON backends (that the user can switch between through environment variables) in Parse::CPAN::Meta::load_json_string.